Production/crab/config/2016/WJet.txt

```diff
@@ -10,3 +10,14 @@ WJetsToLNu_HT-600To800_ext1	     /WJetsToLNu_HT-600To800_TuneCUETP8M1_13TeV-madg
 WJetsToLNu_HT-800To1200_ext1 	 /WJetsToLNu_HT-800To1200_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3_ext1-v2/MINIAODSIM
 WJetsToLNu_HT-1200To2500_ext1	 /WJetsToLNu_HT-1200To2500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3_ext1-v2/MINIAODSIM
 WJetsToLNu_HT-2500ToInf_ext1	 /WJetsToLNu_HT-2500ToInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3_ext1-v2/MINIAODSIM
+W1JetsToLNu_TuneCP5          	 /W1JetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer19UL16MiniAOD-106X_mcRun2_asymptotic_v13-v2/MINIAODSIM
```

Comment:
all other W samples for 2016 are done with TuneCUETP8M1 => no need to produce TuneCP5 for 2016